src/components/StylesFromLiteralsService.js

```diff
@@ -0,0 +1,154 @@
+(function() {
+  goog.provide('ga_styles_from_literals_service');
+
+  var module = angular.module('ga_styles_from_literals_service', []);
+
+  module.provider('gaStylesFromLiterals', function() {
+
+    this.$get = function() {
+
+      function getOlStyleForPoint(options, shape) {
+        if (shape === 'circle') {
+          return new ol.style.Circle(options);
+        } else if (shape === 'icon') {
+          return new ol.style.Icon(options);
+        } else {
+          var shapes = {
+            square: {
```

Comment:
@procrastinatio In ol2 styling can we draw squares and triangles? Would you mind adapting the print so that we can support those styles? Ideally a PR in this one.